sds/src/scanner/mod.rs

```diff
@@ -1434,4 +1444,145 @@ mod test {
         // This should match because "test" is not found, so it's not a false-positive
         assert_eq!(matches.len(), 1);
     }
+
+    mod metrics_test {
```

Comment:
This needs `#[cfg(test)]` so it's only compiled when running tests. (This allows several things such as speeding up compilation when not running tests, and allowing using "dev-dependencies" only during tests)

Reply:
Even if the `mod` declaration is nested in a module that is already `#[cfg(test)]` ?